### PR TITLE
Refactor Messages

### PR DESCRIPTION
**Pull request by jamessinmaojun**

### CODE REVIEW

**Review by chiageng — APPROVED** [review]
LGTM

**Review by jannaleong — APPROVED** [review]
LGTM!!

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java [hunk 1]
```diff
@@ -39,8 +40,16 @@ public EditCommand parse(String args) throws ParseException {
 
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_FIELD);
+
+        // check for missing name
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME)) {
```
Comment (yleeyilin) [inline src/main/java/seedu/address/logic/parser/EditCommandParser.java:45]: Here should check prefix field exists

Comment (yleeyilin) [reply src/main/java/seedu/address/logic/parser/EditCommandParser.java:45]: same for all other classes

**Review by yleeyilin — COMMENTED** [review]
minor comment, otherwise, lgtm !

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java (hunk 1 above)
Comment (jamessinmaojun) [reply src/main/java/seedu/address/logic/parser/EditCommandParser.java:45]: hmmm change to this?
```
if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_FIELD))
```

Comment (yleeyilin) [reply src/main/java/seedu/address/logic/parser/EditCommandParser.java:45]: yes

**Review by Joshy837 — APPROVED** [review]
LGTM

**Pull request merged**